### DISCUSSION TITLE
Google classroom

### DISCUSSION THREAD

**Discussion by blackline0911** [1101-GS4719-C/Python-Community-GS4719-C#642] [General]
https://edu.google.com/workspace-for-education/classroom/

Google 公司建立的一套應用程式，特別為線上教學的老師與學生建立的學習介面。
功能上跟中央大學學生使用的eeclass類似，但是Google class 免費為全球教師提供一個學習環境，且做到保留學生個資等資料不會受到網路影響，是一個公平且公開的學習平台。
可以推薦給想要開授個人課程的老師使用。

**Comment by 11timyyy** [comment]
感覺對老師而言會是一個很棒的工具

**Comment by yuhao112** [comment]
不得不說google 在這方面的東西做的還真周全呢!

**Comment by d26488399** [comment]
高中的時候,學校的外師很愛用

**Comment by 110409538** [comment]
我高中的時候上網課基本上都是使用Google classroom。我覺得統一用同一個應用程式會更加方便。

**Comment by jason79461385** [comment]
以前高中的時候有使用過，但當時老師都只有放一些線上的資源在那裏就是了。所以到後來就都沒什麼去看。

**Comment by awkward-willy** [comment]
Google 的產品真的充斥在我們的生活之中了
如果能建置完全 + 有大量的使用者相信能增加大家線上課的學習成效
當然還是得看大家願不願意打開來看老師放的資源啦

**Comment by stan890425** [comment]
我們值錢有課就是用classroom
他的資料會保存在google裡面
可以多年後再回來看

**Comment by liangyinglly** [comment]
高中時期都是用google classroom繳交作業 方便好上手

**Comment by leon10456** [comment]
老師超愛
